malai-core/org.malai.ts/src/binding/AnonNodeBinding.ts

```diff
@@ -122,12 +127,11 @@ export class AnonNodeBinding<C extends CommandImpl, I extends TSInteraction<D, F
 
     public fsmStops(): void {
         super.fsmStops();
-        if (this.currentCmd !== undefined) {
```

Comment:
why suppressing this.cmd != undefined
?

Reply:
This should be as fsmCancels, no?
`if (this.endOrCancelFct && this.cmd !== undefined) {`

Reply:
Because in super.fsmStops(), this.cmd is always set to undefined